### ISSUE TITLE
Missing manage_(.+).py file

### ISSUE THREAD

**Issue by mielune** [mgodlewski/dockerfiles#3]
Hi,

To use /var/lucterios2/launch_lucterios.sh to make backup/restore, the script check if exists a file named "manage_$DIACAMMA_ORGANISATION.py" in the instances path (/var/lucterios2).
Building an empty file seems to work to launch a restore.

**Comment by mathieupotier** [comment]
Hi,

Can you please provide the exact command you're trying to run, as we can investigate further ?

Regards,

**Comment by mielune** [comment]
I started the docker container to import a backup of an old database of Diacamma asso to help the new management team of this asso to use Diacamma. It will be the same to people who would like to migrate from a “classical” installation to a docker one.
To manage restore/backup we need to use the lucterios sh/psy script in command line. I did it throw a docker exec command. The lucterios script search for each “manage_(.+).py” files in the base directory to find installed databases.
I created an empty one to try the script and it work well for the “restore” function.
The best way will be to create a real “manage” file during the 1st start of the container. 

I hope that this details will help you.


Laurent
